### DISCUSSION TITLE
Some of the Vusers are FINISHED FAILED but tpm count is there.

### DISCUSSION THREAD

**Discussion by shreyapal24** [TPC-Council/HammerDB#630] [Q&A] [answered]
TPCC Hammerdb result says- Jobid has no test result.
A few of the Vusers fail without any error. Can some please help me decipher this?
[tpcc.log](https://github.com/TPC-Council/HammerDB/files/13219762/tpcc.log)

**Comment by sm-shaw** [comment, marked as answer]
If you run the [HammerDB web service](https://www.hammerdb.com/blog/uncategorized/hammerdb-v4-8-new-features-viewing-your-benchmark-results-with-the-hammerdb-web-service/) it will show you automatically generated charts to help you interpret your results. 
For example if you click on transaction count chart, you will see something like this: 
![image](https://github.com/TPC-Council/HammerDB/assets/38044085/0cd591f5-ff6f-4549-8969-3acabc96e392)
So you know that you have a database configuration error, e.g. did it simply just run out of space? that would cause the virtual users to fail. 
you need to look in your system and database logs to see what happened. 
Also note for PostgreSQL there is a graphical metrics viewer that can help you see what your database is waiting for. 
https://www.hammerdb.com/blog/uncategorized/hammerdb-v4-3-new-features-pt1-graphical-metrics-for-postgresql/
